File: C/Algorithm/IfElse.c
#include <stdio.h>

int main(void) {
	int v1, v2;
	printf("Valor 1 :"); scanf("%d", &v1);
	printf("Valor 2 :"); scanf("%d", &v2);
	if (v1 == v2) printf("O valor é igual\n");
	else printf("O valor é diferente\n");
	if (v1 > v2) printf("O valor maior é %d\nO valor menor é %d", v1, v2);
	else printf("O valor maior é %d\nO valor menor é %d", v2, v1);
}

//https://pt.stackoverflow.com/q/329552/101

File: PHP/Syntax/IfElse.php
if ($valor < 10) {
    $total += $valor
} elseif ($total < 20) {
    $total += $valor * 1.1;
} elseif ($total < 30) {
    $total += $valor * 1.2;
} else {
    $total += $valor * 1.3;
}

if ($valor < 10) {
    $total += $valor
}
if ($total < 20) {
    $total += $valor * 1.1;
}
if ($total < 30) {
    $total += $valor * 1.2;
}
if ($total >= 30) {
    $total += $valor * 1.3;
}

if ($valor < 10) {
    $total += $valor
} else {
    if ($total < 20) {
        $total += $valor * 1.1;
    } else {
        if ($total < 30) {
            $total += $valor * 1.2;
        } else {
            if ($total >= 30) {
                $total += $valor * 1.3;
            }
        }
    }
}

//https://pt.stackoverflow.com/a/57792/101
